sematic/types/types/aws/s3.py

```diff
@@ -0,0 +1,22 @@
+# Standard Library
+from dataclasses import dataclass
+
+
+@dataclass
+class S3Bucket:
+    """
+    Basic type to describe an S3 bucket.
+    """
+
+    region: str
```

Comment:
Is region strictly required? Might it be an `Optional` field? When people configure the S3 bucket for storage, they don't have to provide a region for it.